packages/functions/src/graphql/types/article.ts

```diff
@@ -1,4 +1,4 @@
-import { Article } from "../../../../core/src/article";
+import * as Article from "../../../../core/src/article";
```

Comment:
I used "import * as Article" instead of "import { Article }," and then an error occurred when I ran "yarn dev". The error is: 

```
jiahao@jiahaodeMacBook-Air-2 graphql-sst-sample % yarn dev
yarn run v1.22.19
$ npm run env:local sst dev

> graphql-sst-sample@0.0.0 env:local
> env-cmd -f .env.local -- sst dev

SST v2.16.3  ready!

➜  App:     graphql-sst-sample
   Stage:   develop
   Console: https://console.sst.dev/graphql-sst-sample/develop

✖  Pothos: Failed to extract schema:
   Entity is not defined

✔  Deployed:
   Database
   Api
   API: https://hogt05y159.execute-api.ap-northeast-1.amazonaws.com
   Web
```